src/oscar/test/utils.py

```diff
@@ -3,11 +3,15 @@
 from django.contrib.sessions.backends.db import SessionStore
 from django.core.signing import Signer
 from django.test import RequestFactory as BaseRequestFactory
-from webob.compat import bytes_
-from webob.cookies import _unquote
 
 from oscar.core.loading import get_class, get_model
 
+try:
+    # Python 3
+    from http.cookies import _unquote
+except ImportError:
+    from Cookie import _unquote
```

Comment:
Does that make sense to move to compat?

Reply:
Yep, have moved to compat and added a docstring.